README.md

```diff
@@ -29,6 +29,7 @@ module "terraform-dome9-awp-aws" {
   # e.g:
```

Comment:
# The scan mode for the AWP. Valid values are "inAccount" or "saas".
  
  Add centralized and sub values

Reply:
done